helm/kubernetes-metrics-server-chart/templates/cluster-role.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:{{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
```

Comment:
Do we need `app` and `k8s-app`?

I added that in the ingress controller chart because i needed it to run the temp resources in parallel. Maybe it can be simpler here.

Reply:
not sure if k8s-app might still be used for official addons, so I'd leave it in

Reply:
Removed, not needed

Reply:
👍 Cool, nice to simplify.